src/main/resources/application.yaml

```diff
@@ -84,6 +84,9 @@ etcos:
     username: et.service@hmcts.net
     password: Pa55word11
 
+case-details-url:
+  exui: ${EXUI_CASE_DETAILS_URL:http://localhost:3455/cases/case-details/}
```

Comment:
default value here should be using 3000 instead of 3455

Reply:
Updated